data/mods/Magiclysm/Spells/classless.json

```diff
@@ -156,5 +156,50 @@
     "base_energy_cost": 675,
     "energy_source": "MANA",
     "spell_class": "NONE"
+  },
+  {
+    "id": "acid_resistance",
+    "type": "SPELL",
+    "name": "Acid Resistance",
+    "description": "Protects the user from acid.",
+    "valid_targets": [ "none" ],
+    "min_range": 0,
+    "max_range": 0,
+    "min_damage": 1,
+    "max_damage": 1,
+    "effect": "spawn_item",
+    "effect_str": "acid_res_aura",
+    "energy_source": "MANA",
+    "spell_class": "NONE",
+    "flags": [ "CONCENTRATE", "NO_LEGS" ],
+    "difficulty": 4,
+    "max_level": 15,
+    "base_casting_time": 200,
+    "base_energy_cost": 150,
+    "min_duration": 84000,
+    "max_duration": 360000,
+    "duration_increment": 18000,
+    "learn_spells": { "acid_resistance_greater": 15 }
+  },
+  {
+    "id": "acid_resistance_greater",
+    "type": "SPELL",
+    "name": "Greater Acid Resistance",
+    "description": "Protects the user from acid.",
+    "valid_targets": [ "none" ],
+    "min_range": 0,
+    "max_range": 0,
+    "min_damage": 1,
+    "max_damage": 1,
+    "effect": "spawn_item",
+    "effect_str": "acid_res_aura_greater",
+    "energy_source": "MANA",
+    "spell_class": "NONE",
+    "flags": [ "CONCENTRATE", "NO_LEGS" ],
+    "difficulty": 7,
+    "max_level": 1,
```

Comment:
i don't see a reason not to have a max level with duration increase.

Reply:
It's max level 1 so it doesn't look weird casting a level 0 spell forever.

Reply:
sorry, i was clear. i mean you could have a max level of 5 or 10 and it would be perfectly fine